### ISSUE TITLE
Add Strafunski-StrategyLib to stackage

### ISSUE THREAD

**Issue by alanz** [jkoppel/Strafunski-StrategyLib#9]
I am in the process of adding HaRe to stackage, this will require Strafunsky-StrategyLib to be there too.

Please add it with yourself as a maintainer, otherwise I can add it too and take responsibility for being postman if anything ever breaks.

See https://github.com/fpco/stackage/pull/906

**Comment by jkoppel** [comment]
I am not comfortable committing seriously enough to add Strafunski to
stackage. Would you like to take over as maintainer?

**Comment by alanz** [comment]
Hi James

If HaRe is the only user then that may make sense, in which case I would be
happy to do it.

Alan

**Comment by jkoppel** [comment]
Okay then. What do you need me to do to transfer ownership?

**Comment by alanz** [comment]
You should be able to go to https://github.com/jkoppel/Strafunski-StrategyLib/settings and scroll down into the red zone, and do it there. That is how I did it with language-javascript and hjsmin

**Comment by jkoppel** [comment]
I think I'd prefer if you just simply fork the repo

**Comment by alanz** [comment]
Ok.

Alan

**Comment by alanz** [comment]
Can you please add AlanZimmerman to the maintainers group for this on
hackage.

**Comment by jkoppel** [comment]
Uhhh.....what? To my knowledge, you do that just by uploading a new version
yourself with a new .cabal file

**Comment by alanz** [comment]
Nope, not on the new hackage. Its been like that for a few months

http://hackage.haskell.org/package/Strafunski-StrategyLib/maintain

I get forbidden if I go here

**Comment by jkoppel** [comment]
Okay, user AlanZimmerman has been added

**Comment by alanz** [comment]
Thanks. You can relax now :)
